api/base/pagination.py

```diff
@@ -39,3 +83,34 @@ def get_paginated_response(self, data):
             ])),
         ])
         return Response(response_dict)
+
+    def paginate_queryset(self, queryset, request, view=None):
+        """
+        Custom pagination of queryset. Returns page object or `None` if not configured for view.
+
+        If this is an embedded resource, returns first page, ignoring query params.
+        """
+        if request.parser_context['kwargs'].get('is_embedded'):
+            self._handle_backwards_compat(view)
```

Comment:
Yes. The docstring for that method says that it's specifically for backwards compat with DRF versions < 3.1, which we're not concerned with.